src/templates/lesson.jsx

```diff
@@ -31,7 +31,7 @@ export default class LessonTemplate extends React.Component {
           </HeaderContainer>
           <ToCContainer>
             <TableOfContents
-              posts={this.props.data.tableOfContents}
+              chapters={this.props.data.tableOfContents.chapters}
```

Comment:
We are not interested in anything else but the chapters anyhow so we can already filter for them here.